gamestonk_terminal/forex/fx_controller.py

```diff
@@ -94,16 +90,8 @@ def print_help(self):
                 "    reddit        search reddit for posts about the loaded instrument"
             )
             print(
-                "    edasummary    brief summary statistics using exploratory data analysis"
-            )
-            print("    edarolling    rolling mean and std deviation")
-            print(
-                "    edadecompose  decomposition in cyclic-trend, season, and residuals"
-            )
-            print(
-                "    edacusum      detects abrupt changes using cumulative sum algorithm"
+                "    eda           exploratory data analysis,	 e.g.: decompose, cusum, residuals analysis"
```

Comment:
Could you add a `>`?

This is to signal the user that this is a menu command :)